src/Knp/DictionaryBundle/Command/DictionaryDumpCommand.php

```diff
@@ -83,12 +85,17 @@ protected function execute(InputInterface $input, OutputInterface $output)
     private function getDictionariesDetail($dictionaryName = null)
     {
         $tableRows = [];
-        /** @var Dictionary $dico */
-        foreach ($this->registry->all() as $dico) {
-            if (!\is_null($dictionaryName) && $dictionaryName === $dico->getName() || \is_null($dictionaryName)) {
-                $tableRows[] = ["<fg=cyan>{$dico->getName()}</fg=cyan>"];
+
+        if (!\is_null($dictionaryName)) {
```

Comment:
Maybe make code as follow:
```
$dico = \is_null($dictionaryName) ? $this->registry->all() : $this->registry->get($dictionaryName);
foreach ($dico as $key => $value) {
    $tableRows[$dico->getName()][] = [$key,$value];
}
```
This will avoid duplicate code line 90 & line 97 in foreach